.travis.yml

```diff
@@ -14,12 +13,14 @@ cache:
 
 matrix:
   include:
-    - php: 5.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.0
-      env: SYMFONY_VERSION=2.7.*
-    - php: 7.0
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.* COMPOSER_FLAGS="--prefer-lowest"
+    - php: 7.1
       env: SYMFONY_VERSION=2.8.*
+    - php: 7.1
+      env: SYMFONY_VERSION=2.8.*
```

Comment:
two times 2.8.*  ?

Reply:
erm. whatever i did, thanks for spotting, deleting this